src/particles/pusher/PlasmaParticleAdvance.cpp

```diff
@@ -17,7 +17,12 @@ AdvancePlasmaParticles (PlasmaParticleContainer& plasma, Fields & fields,
                         const bool do_update, const bool do_shift, int const lev,
                         PlasmaBins& bins)
 {
-    HIPACE_PROFILE("UpdateForcePushParticles_PlasmaParticleContainer()");
+    char str[38] = "UpdateForcePushParticles_Plasma(    )";
```

Comment:
Please use `std::string` and `.at()` for such endeavors :)

https://en.cppreference.com/w/cpp/string/basic_string/at